app/assets/javascripts/misc/pw-strength.js

```diff
@@ -27,9 +27,15 @@ function getFeedback(z) {
   if (!z || z.score > 2) return '';
 
   const { warning, suggestions } = z.feedback;
+  const lookup = function(str) {
```

Comment:
Functionally it's the same, but in this case I do think it's a little easier to follow whats going on if it's just a function so I just updated this.